src/Evaluation/Service/EvaluationService.ts

```diff
@@ -0,0 +1,55 @@
+import { StorageService, LocalStorage } from '../../localStorageService';
+
+export type EvaluationType =
+  | "Assignment"
+  | "Mid Exam"
+  | "Quiz"
+  | "Project"
+  | "Practical Lab"
+  | "Final Exam";
+
+export interface Evaluation {
+  course: string;
+  title: string;
+  type: EvaluationType;
+  weight: number;
+  dueDate: Date;
+  instructor: string;
+  campus: string;
+}
+
+const EVALUATION_DATA_STORAGE_KEY = 'Evaluation_Data_Storage';
+
+export class EvaluationService {
+  constructor(
+    private storageService: StorageService
+  ) {}
+
+  static createDefault(): EvaluationService {
+    const storage = new LocalStorage();
+    return new EvaluationService(storage);
+  }
```

Comment:
don't use static. this can simply be done using the constructor.

```suggestion

  storageService: StorageService;
  
  constructor(storageService: StorageService) {
    this.storageService = storageService;
  }
```

everything else can reference `this.storageService`